pkgs/stdenv/generic/setup.sh

```diff
@@ -1022,6 +1022,15 @@ configurePhase() {
             fixLibtool "$i"
         done
 
+        # Make sure `find` is the full `find` from findutils, rather
+        # than from busybox or some other implementation.  This is
+        # analogous to `shellcheck shell=bash` at the top of this
+        # file.
+        if (( $(find --version | grep -q 'GNU findutils') != 0 )); then
+            echo 'the find in $PATH is not findutils'
```

Comment:
```suggestion
            echo 'The find contained in $PATH is not the GNU variant from the findutils package.'
```
We should give some more explanation to easily convey what the problem is.